### ISSUE TITLE
Feature request: Save column width resizing 

### ISSUE THREAD

**Issue by wisemike2** [stefankueng/grepWin#80]
When enlarging "Search results" columns like "name" or "path" to see more text in each, it gets reset each time a new result is performed.

However after the scanning of files ends, it automatically enlarges the columns to fit.  So, adding an option to save the user preference of the column width, would fix this issue.  Thanks.

**Comment by RoyiAvital** [comment]
Could we also have option to reset `grepWin` to its initial default state (As a clean install)?
Namely all buttons, text, settings, etc... are reset.

**Comment by Gliese-667** [comment]
I'm not sure if this has been addressed in the past or not, but this issue seems to be present in Windows 11 as well. I don't specifically remember it happening before I updated, but I may not have noticed. I frequently find the Text column shrinking itself rather than staying put. I also noticed if you click on one of the banners to sort a column, the Text column slowly works itself left despite the search results not changing.

Thank you for everything you do!!

**Comment by mrx23dot** [comment]
Also please maximize width, there is empty space after path

![image](https://github.com/stefankueng/grepWin/assets/4163396/48ff3482-67b7-4355-a725-454a8e1a2e3a)

**Comment by glenneroo** [comment]
Yes this would be amazing to save column widths, it is very frustrating when performing lots of searches esp. with the Content window.

Related: When you click a column header to sort, it resizes all column widths which is very frustrating since the auto-sizing is often completely messed up, and I have to resize every column width manually.
